lib/spack/spack/schema/config.py

```diff
@@ -94,6 +94,14 @@
             "additional_external_search_paths": {"type": "array", "items": {"type": "string"}},
             "binary_index_ttl": {"type": "integer", "minimum": 0},
             "aliases": {"type": "object", "patternProperties": {r"\w[\w-]*": {"type": "string"}}},
+            "includes": {
```

Comment:
Can we please call this `include:` to match the existing env `include:`?https://spack.readthedocs.io/en/latest/environments.html#included-configurations